.github/workflows/php.yml

```diff
@@ -0,0 +1,37 @@
+name: PHP Composer
+
+on:
+  push:
```

Comment:
Can we remove "on push" keeping only "on pull request".

Reply:
Sure, updated.